### DISCUSSION TITLE
Can we download the previous version of BORIS for MacOS?

### DISCUSSION THREAD

**Discussion by NalinaAiem** [olivierfriard/BORIS#699] [Q&A]
Hi,

My colleague wants to download BORIS on her MacBook Pro M1. As v.8.22.6 is not yet available for MacOS, can we download the previous version of BORIS to use for the time being? If so, could use please provide us the download link?

Thank you!

**Comment by olivierfriard** [comment]
Hi Nalina,
my advice is to NOT use an old version of BORIS because it contains bugs and limited features.
Try to use an alternative way to run BORIS on your Mac. See http://www.boris.unito.it/pages/download_mac

**Comment by outergrace** [comment]
Hi Nalina,

Wondering if you were successful downloading Boris onto your MacOS and if so, how you were able to do that? Thanks,

Grace

**Comment by olivierfriard** [comment]
Hi,
I just released a version of BORIS that should run natively on macOS with some limitations described on https://www.boris.unito.it/download_mac.
This is an experimental version so test it and let me know

**Comment by NalinaAiem** [comment]
Hi,

This is a great news! Thanks so much!

Best,
Nalina
